staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/fairqueuing.go

```diff
@@ -254,10 +231,16 @@ func (qs *queueSetImpl) ChooseQueueIdx(hashValue uint64, handSize int) int {
 	// TODO(aaron-prindle) currently a lock is held for this in a larger anonymous function
 	// verify that makes sense...
 
+	bestQueueIdx := -1
+	bestQueueLen := 2147483647
```

Comment:
math.MaxInt32?

Reply:
This way it is an integer and no conversion is required when doing the comparison

Reply:
Oh, I see that constant is actually untyped.  So the revised code has an unnecessary typecast.  Oh well.